includes/polling/os/f5.inc.php

```diff
@@ -18,3 +18,66 @@
     data_update($device, 'bigip_apm_sessions', $tags, $fields);
     $graphs['bigip_apm_sessions'] = true;
 }
+
+
+
+$sysClientsslStatTotNativeConns = snmp_get($device, 'sysClientsslStatTotNativeConns.0', '-OQv', 'F5-BIGIP-SYSTEM-MIB');
+$sysClientsslStatTotCompatConns = snmp_get($device, 'sysClientsslStatTotCompatConns.0', '-OQv', 'F5-BIGIP-SYSTEM-MIB');
+if (is_numeric($sysClientsslStatTotNativeConns) && is_numeric($sysClientsslStatTotCompatConns)) {
```

Comment:
Could you use snmp_get_multi instead of multiple snmp_get ?

Reply:
Yep. Done in  commit https://github.com/librenms/librenms/pull/9883/commits/8187008f91037b2113f7373220e9ab5070b20126

Reply:
Good start :) But the goal is to have only one snmp-multi-get collecting all the data. And then you can add the data to the RRDs.
And if you feel optimizing, you could write a loop that does add the variables to the RRDs instead of duplicating very similar code for each variable.

Reply:
Thanks for the hint. :) I'll rewrite the entire poller for f5 devices in the next days including the apm.